packages/aws-cdk-lib/aws-codedeploy/lib/private/utils.ts

```diff
@@ -31,12 +31,16 @@ export function arnForDeploymentConfig(name: string, resource?: IResource): stri
   });
 }
 
-export function renderAlarmConfiguration(alarms: cloudwatch.IAlarm[], ignorePollAlarmFailure: boolean | undefined, removeAlarms = true):
-CfnDeploymentGroup.AlarmConfigurationProperty | undefined {
+export function renderAlarmConfiguration(
+  alarms: cloudwatch.IAlarm[],
+  ignorePollAlarmFailure: boolean | undefined,
+  removeAlarms = true,
+  ignoreAlarmConfiguration: boolean = false,
```

Comment:
```suggestion
  ignoreAlarmConfiguration = false,
  removeAlarms = true,
```
I think it's better/clearer to keep the feature flag parameter last.